package.json

```diff
@@ -82,12 +82,12 @@
   "dependencies": {
     "@material-ui/lab": "^4.0.0-alpha.56",
     "@popperjs/core": "^2.4.0",
+    "@superdesk/primereact": "^5.0.2-3",
```

Comment:
Is it safe to go to version 6x?

Reply:
I don't think so. It's a major release so we would need to re-test all components that we use.